tests/test_aamp_mmotifs.py

```diff
@@ -0,0 +1,248 @@
+import numpy as np
+import numpy.testing as npt
+import naive
+import pytest
+
+from stumpy.aamp_mmotifs import aamp_mmotifs
+from stumpy import config
+
+
+test_data = [
+    np.array(
+        [
+            [5.2, 0.1, 3.5, 3.4, 7.1, 9.8, 3.7, 5.0, 2.1, 4.3, 7.5, 6.8, 8.0, 8.1, 1.2],
+            [
+                7.3,
+                3.2,
+                5.0,
+                9.1,
+                8.2,
+                7.3,
+                4.8,
+                8.2,
+                10.0,
+                0.0,
+                4.1,
+                3.2,
+                2.3,
+                0.1,
+                1.4,
+            ],
+            [6.2, 7.6, 7.6, 8.4, 1.1, 5.9, 9.2, 8.5, 9.3, 4.6, 3.5, 0.0, 3.1, 5.3, 0.9],
+            [
+                0.1,
+                1.3,
+                3.0,
+                2.1,
+                6.2,
+                1.3,
+                9.5,
+                10.0,
+                1.8,
+                2.0,
+                2.1,
+                5.2,
+                1.3,
+                0.5,
+                4.3,
+            ],
+        ]
+    )
+]
+
+
+def test_aamp_mmotifs_default_parameters():
```

Comment:
Perfect!

Reply:
I don't know if you notice this but after we take out the chunk of code for `T`, the test function(s) are all super simple and it is easy to digest what is happening. This little bit of refactoring, cleans up the code and everything can be read without scrolling. This is exactly how a "good" unit test should look and this is about as close to "perfection" as I could expect, which makes my job as a reviewer really, really easy. The rest of the unit tests look equally as great too! Excellent work!

"Future Sean" thanks you as well. 😄